docs/rules/Moq1410.md

```
@@ -0,0 +1,71 @@
+# Moq1410: Explicitly set the Strict mocking behavior
+
+| Item     | Value   |
+| -------- | ------- |
+| Enabled  | True    |
+| Severity | Info    |
+| CodeFix  | True    |
```

Markdownlint (0.37.0)

Tables should be surrounded by blank lines

(MD058, blanks-around-tables)